pegasus/sites.v3/code.org/public/ai/how-ai-works.haml

```diff
@@ -32,6 +32,8 @@ theme: responsive_full_width
         =hoc_s(:call_to_action_explore_lessons)
     .clear
 
+= view :dance_party_ai_skinny_banner
```

Comment:
Added this banner here:

![ai_how-ai-works](https://github.com/code-dot-org/code-dot-org/assets/9256643/25f3178e-79dc-4e41-87ab-50dbbc48fb3d)